lib/tcr/version.rb

```diff
@@ -1,3 +1,3 @@
 module TCR
-  VERSION = "0.3.0"
+  VERSION = "1.0.0"
```

Comment:
I'll set versions as part of the release process, don't worry about doing so in branches.